README.md

```diff
@@ -117,36 +119,47 @@ badges:
 [![Powered by Flame](https://img.shields.io/badge/Powered%20by-%F0%9F%94%A5-orange.svg?style=for-the-badge)](https://flame-engine.org)
 ```
 
-## Contributing
-
-__Warning__: We are working on bringing Flame to its first stable version, updates on `0.x` versions
-are frozen, except for crucial bug fixes. If you want to contribute to that version, please be
-mindful of that, and use the `master-v0.x` branch. For contributions for v1, your PR must point to
-the `main` branch. If in doubt, make sure to talk about your contribution to the team, either via an
-issue or [Discord](https://discord.gg/pxrBmy4).
 
-Any help is appreciated! Comments, suggestions, issues, PRs.
+## Contributing
 
 Have you found a bug or have a suggestion of how to enhance Flame? Open an issue and we will take a
 look at it as soon as possible.
 
-Do you want to contribute with a PR? PRs are always welcome, just be sure to create it from the
-orrect branch (see above) and follow the [checklist](.github/pull_request_template.md) which will
+Do you want to contribute with a PR? PRs are always welcome, just make sure to create it from the
+correct branch (main) and follow the [checklist](.github/pull_request_template.md) which will
 appear when you open the PR.
 
+If in doubt, make sure to talk about your contribution to the team, either via an issue or
+[Discord](https://discord.gg/pxrBmy4).
+
+
 ## Getting started
 
-A collection of guides can be found [here](./tutorials). Note that these tutorials are based on the
-main branch. To make sure you are looking at the tutorials that work with your current version,
-select the correct version tag.
+A simple tutorial to get started can be found on
+[tutorials.flame-engine.org](https://tutorials.flame-engine.org) and examples of most features in
+Flame can be found on [examples.flame-engine.org](https://examples.flame-engine.org). To access the
+code for each example, press `< >` in the top right corner.
+
+
+### External tutorials
```

Comment:
It is too bad that the GitHub markdown doesn't support any include directive, because otherwise the `TUTORIALS.md` file will probably go the same way as our `FAQ.md` file did. And also, it's nice to have the links directly there so that you don't have to click on another link first to go to the links. But if everyone agrees that a separate file is the way to go I'll create that.